src/Microsoft.IdentityModel.Tokens/Validation/Validators.IssuerSigningKey.cs

```diff
@@ -18,16 +18,14 @@ namespace Microsoft.IdentityModel.Tokens
     /// <param name="signingKey">The security key to validate.</param>
     /// <param name="securityToken">The <see cref="SecurityToken"/> that is being validated.</param>
     /// <param name="validationParameters">The <see cref="TokenValidationParameters"/> to be used for validating the token.</param>
```

Comment:
Update <see cref="TokenValidationParameters"/> to <see cref="ValidationParameters"/>.